components/sysutils/glances/glances.desktop

```diff
@@ -0,0 +1,67 @@
+[Desktop Entry]
```

Comment:
Where this file comes from?  It lacks the copyright/license header.

Reply:
I don't know much about Mate. I copied /usr/share/applications/htop.desktop to glances.desktop and configured. Now the htop package self does not contain any htop.desktop, so I assume this was generated by Mate??
It would be even easier if I could generate a .desktop for glances.

Reply:
Then, maybe, it would be better to do not include this file?  Since Glance seems to be a terminal utility we do not need desktop file for it, IMHO.

Reply:
BTW: This package is also a candidate as we don't want to have /usr/bin/glances-3.9